server/@types/dto/index.d.ts

```diff
@@ -0,0 +1,14 @@
+declare module 'dto' {
+  export interface CreateGoalDto {
+    prisonNumber: string
+    title: string
+    reviewDate: Date
+    steps: [AddStepDto]
```

Comment:
I don't know the answer to this one ... but is that the right syntax for declaring the array vs. instantiating the array?
I've seen in other places (in this codebase) where we use `Array<type>` , so I wonder if your line 6 should be:
```
    steps: Array<AddStepDto>
```

I could be wrong, but to me it looks like what you have is the instantiation of an array with 1 element which is the `AddStepDto` type (rather than an array of `AddStepDto` instances)

Like I said, I'm no expert here,  but thats what it looks like to me?

Reply:
I think you might be right 👍

Reply:
Updated 👍